gamestonk_terminal/cryptocurrency/coinpaprika/README.md

```diff
@@ -0,0 +1,321 @@
+# Coinpaprika
+
+This menu aims to explore crypto world with [Coinpaprika](#https://coinpaprika.com/)
+The usage of the following commands along with an example will be exploited below.
+
+Coinpaprika research platform was founded in April 2018. Coinpaprika mission is to provide comprehensive, reliable, transparent
+and objective access to information about crypto projects from all around the world. [Source: See https://coinpaprika.com/about-us/]
+
+
+[COIN](#COIN)
+* [find](#find)
+  * find similar coin by coin name,symbol or id
+* [load](#load)
+  * load a given coin, you can use either coin symbol or coin id
+* [clear](#clear)
+  * clear loaded coin
+* [chart](#chart)
+  * plot the loaded crypto chart
+* [ta](technical_analysis/README.md)
+  * open technical analysis menu
+* [basic](#basic)
+  * show coin basic information
+* [ps](#ps)
+  * show coin price, supply, market cap related metrics.
+* [mkt](#mkt)
+  * show all markets for loaded coin
+* [ex](#ex)
+  * show all exchanges where loaded coin is listed
+* [twitter](#twitter)
+  * show up to 50 last tweets for loaded coin
+* [events ](#events )
+  * show events related to loaded coin (for most of cryptocurrencies there is no data)
```

Comment:
The (comment) probably is not needed here, so that this matches the short/long descriptions shown in the terminal